crates/ruma-common/src/events/content.rs

```diff
@@ -120,7 +120,7 @@ pub trait MessageLikeEventContent: EventContent<EventType = MessageLikeEventType
 /// Content of a redacted message-like event.
 pub trait RedactedMessageLikeEventContent: MessageLikeEventContent + RedactedEventContent {}
 
-/// Content of a redacted state event.
+/// Content of a non-redacted state event.
 pub trait StateEventContent: EventContent<EventType = StateEventType> {
```

Comment:
Well, this trait is implemented by redacted event content types as well.. (as the supertraits on `RedactedStateEventContent` show)